tools/logging.config

```diff
@@ -0,0 +1 @@
+{"Enable":false,"SendToLog":false,"URI":"","Name":"Primary","GUID":"06411ccb-ccc0-4ba6-8c9b-bc67efac2e5f","FilePath":"","UserName":"","Password":"","MinLogLevel":3,"ReportHistoryLevel":3}
```

Comment:
can the GUID be hardcoded here?

Reply:
I've just tested a blank field for the GUID, and it works. Since we are disabling, it doesn't matter. Let's leave it blank

Reply:
Done